ansible_ai_connect/users/management/commands/tests/test_generate_users_trials_reports.py

```diff
@@ -55,6 +69,19 @@ def call_command(*args, **kwargs):
         )
         return out.getvalue()
 
+    @staticmethod
+    def call_command_with_argv(argv):
+        """
+        Call command with parsing/converting/mapping of argv identically to invocation from the CLI.
+        :param argv:
+        :return:
+        """
+        command = Command()
+        parser = command.create_parser("manage", "generate_users_trials_reports")
+        options = parser.parse_args(argv)
```

Comment:
I believe we can split this up and have one test to validate the parsing and one to validate the behaviour with different set of `options` structure.